widget.py

```diff
@@ -196,11 +235,11 @@ def udp_receive_socket_data(self):
             message_bytes = data[2:]
             header = packet_spec.parse_packet_header(header_bytes)
             message = packet_spec.parse_packet_message(header, message_bytes)
-            #Actuator state handling
-            if(header.sub_type == packet_spec.TelemetryPacketSubType.ACT_STATE):
-                self.updateActState(message)
-            else:
-                self.plot_point(header, message)
+            self.plot_point(header, message)
+
+            #If we want to recording data
+            if self.ui.recordingToggleButton.isChecked():
+                self.file_out.write(datagram)
```

Comment:
Can you revert this change to how it was previously? I think it's a better idea these changes it separate PRs

Reply:
I just change it back already. You can have a look